src/removeDuplicates.md

```diff
@@ -0,0 +1,17 @@
+# Remove duplicates
+Remove Duplicate from an Array
+
+## Result
+```
+  removeDup([1,2,1,3]); // [1,2,3]
```

Comment:
rename to removeDuplicates or something readable

💭 Always write variable names readable